Prog/Neighborhood change/4.1_prediction_OTR.R

```diff
@@ -0,0 +1,338 @@
+setwd("C:/Users/Ysu/Box/Greater DC/Projects/DMPED Housing Assessment 2024/Task 2 - Nbrhd Change and Displacement Risk Assessment/Data collection/")
+
+housingmarket <- read_csv("Clean/housingmarket.csv") 
+
+data <- read_sas("W:/Libraries/Realprop/Data/sales_sum_tr20.sas7bdat")
+#tring the OTR sales data for median home value
+
+OTR_sales <- data %>% 
+  rename(GEOID=Geo2020) %>% 
+  select(GEOID, mprice_tot_1999,mprice_tot_2000,mprice_tot_2001, mprice_tot_2012,mprice_tot_2011,mprice_tot_2010, mprice_tot_2013,mprice_tot_2022,mprice_tot_2021,mprice_tot_2023) %>% 
+  # filter(is.na(mprice_tot_2000)) #8 missing in 2022 #13 missing in 2012 #12 missing in 2000
+  mutate(GEOID=as.numeric(GEOID))
+
+#use the OTR data instead for home value
+housingmarket <- housingmarket %>% 
+  left_join(OTR_sales,by=c("GEOID")) %>% 
+  mutate(medianhome_2000_2020=mprice_tot_2000,
+         medianhome_2012_2020=mprice_tot_2012,
+         medianhome_2022=mprice_tot_2022) %>% 
+  # filter(is.na(medianhome_2022)|is.na(medianhome_2012_2020)) %>%
+  # select(GEOID, NBH_NAMES, total_hh_2022, medianhome_2000_2020,medianhome_2012_2020,medianhome_2022,mprice_tot_1999,mprice_tot_2001, mprice_tot_2011,mprice_tot_2021) %>%
+  mutate(medianhome_2000_2020=ifelse(GEOID=="11001004702",165000,medianhome_2000_2020),
+         medianhome_2022=ifelse(GEOID=="11001005602",1039500,medianhome_2022),
+         medianhome_2012_2020=ifelse(GEOID=="11001007401",178250,medianhome_2012_2020),
+         medianhome_2022=ifelse(GEOID=="11001007401",500000,medianhome_2022),
+         medianhome_2012_2020=ifelse(GEOID=="11001007401",207250,medianhome_2012_2020),
+         medianhome_2012_2020=ifelse(GEOID=="11001009602",284900,medianhome_2012_2020)) %>% #use nearest year sales data if available
+  # filter(is.na(medianhome_2022)|is.na(medianhome_2012_2020)|is.na(medianhome_2000_2020)) %>%
+  # select(GEOID, NBH_NAMES, total_hh_2022, medianhome_2000_2020,medianhome_2012_2020,medianhome_2022,mprice_tot_1999,mprice_tot_2001, mprice_tot_2013,mprice_tot_2021,mprice_tot_2023) %>%
+  # mutate(medianhome_2000_2020=ifelse(medianhome_2000_2020==0, NA, medianhome_2000_2020),
+  # medianhome_2012_2020=ifelse(medianhome_2012_2020==0, NA, medianhome_2012_2020)) %>%
+  filter(!is.na(medianhome_2022)& !is.na(medianhome_2012_2020) & !is.na(medianhome_2000_2020))
+  
+neighborhoodname <- read_csv("Clean/neighborhood_tract.csv")
+
+lowincome <- read_csv("Clean/lowincome_pop.csv")
+
+raceethnicity <- read_csv("Clean/race_ethnicity.csv")
+
+neighborhoodtype_OTR <- read_csv("Clean/neighborhoodtype_homevalueOTR.csv") %>% 
+  select(GEOID, neighborhoodtype, NBH_NAMES)
+
+vacancy <- read_csv("Clean/vacancy.csv") %>% 
+  mutate(GEOID=geoid) %>% 
+  select(GEOID, year, vacancyrate) %>% 
+  filter(year==2012|year==2022) %>% 
+  mutate(year=paste0("vacancy_", as.character(year))) %>% 
+  spread(key=year,value=vacancyrate) 
+
+distance <- read_csv("Clean/distance_downtown.csv")
+
+lowincjobs <- read_csv("Clean/lowincome_jobs.csv")
```

Comment:
Where do lowincome_jobs.csv and HUD_subsidy.csv come from? Please document the sources and, if necessary, processes for compiling these data.

Reply:
vacancy https://github.com/NeighborhoodInfoDC/DMPED/pull/127/files
job and hud data https://github.com/NeighborhoodInfoDC/DMPED/pull/129/files